webapp/src/js/services/db-sync.js

```diff
@@ -49,9 +49,9 @@ angular
     const DIRECTIONS = [
       {
         name: 'to',
+        filter: true,
         options: {
           checkpoint: 'source',
-          filter: readOnlyFilter
```

Comment:
I fail to see the reason why the filter was moved from here to the `replicate` function.

Reply:
Ugh. A test was failing implying the `filter` wasn't being supplied and I thought the Object.assign had broken it, but I think I misread the error and it's passing now. Reverted.